src/workloads/query/GroupLikeDistinct.yml

```diff
@@ -0,0 +1,161 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  This workloads covers $group queries with distinct-like semantics, meaning that only a single document
```

Comment:
Should we add here $group with covered $min/$max accumulators? Technically speaking, these _could_ use distinct scan, we just don't do the optimization currently